lib/config-generator.js

```diff
@@ -220,14 +220,32 @@ class ConfigGenerator {
     }
 
     buildRulesConfig() {
+        const applyRuleConfigurationCallback = (name, defaultRules) => {
+            if (!(name in this.webpackConfig.loaderConfigurationCallbacks)) {
+                throw new Error(`Loader "${name}" is not configurable. Valid loaders are "${Object.keys(this.webpackConfig.loaderConfigurationCallbacks).join('", "')}".`);
+            }
+
+            return applyOptionsCallback(this.webpackConfig.loaderConfigurationCallbacks[name], defaultRules);
+        };
+
+        const applyRuleConfigurationCallbacks = (names, defaultRules) => {
```

Comment:
As noted before, the whole alias-thingy could probably be handled directly inside of `applyRuleConfigurationCallback` to avoid calling multiple callbacks for a single rule.